microbench/main/java/com.twitter.hpack.microbench/AbstractMicrobenchmarkBase.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright 2015 Twitter, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.twitter.hpack.microbench;
```

Comment:
does it also need to be moved under src/test like in netty?

On Tue, Apr 28, 2015 at 1:26 PM, Nathan Mittler <notifications@github.com>
wrote:

> In
> microbench/main/java/com.twitter.hpack.microbench/AbstractMicrobenchmarkBase.java
> <https://github.com/twitter/hpack/pull/24#discussion_r29284441>:
>
> > +/*
> > + * Copyright 2015 Twitter, Inc.
> > + *
> > + * Licensed under the Apache License, Version 2.0 (the "License");
> > + * you may not use this file except in compliance with the License.
> > + * You may obtain a copy of the License at
> > + *
> > + *     http://www.apache.org/licenses/LICENSE-2.0
> > + *
> > + * Unless required by applicable law or agreed to in writing, software
> > + * distributed under the License is distributed on an "AS IS" BASIS,
> > + * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > + * See the License for the specific language governing permissions and
> > + * limitations under the License.
> > + */
> > +package com.twitter.hpack.microbench;
>
> @jpinner <https://github.com/jpinner> oof, good catch! I've replaced
> com.twitter.hpack.microbench with com/twitter/hpack/microbench.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twitter/hpack/pull/24/files#r29284441>.
>